CIP-xxxxx/README.md

```diff
@@ -0,0 +1,208 @@
+---
+CIP: ???
+Title: Web-Wallet Bridge - Plutus wallets
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors: NA
+Discussions:
+Created: 2023-10-12
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This document describes a CIP-30 extension allowing webpages (i.e., DApps) to interface with Cardano Plutus wallets. This interface is a work in progress, and new versions are expected to be included as the ecosystem evolves. 
+
+## Motivation: why is this CIP necessary?
+
+In order to facilitate future DApp development, we will need a way for DApps to communicate with Plutus wallets, given the unique complexities of Plutus-based addresses. Special provisions need to be made to make the connector compatible with them.  
```

Comment:
needs more explicit info about what "Plutus wallet" actually means.

Reply:
Added a section explaining Plutus wallets and a few possible implementation of them